src/pages/partners/affiliate-ib/_deriv-affiliate-programme.tsx

```diff
@@ -211,14 +203,14 @@ const DerivAffiliateProgramme = () => {
                     <CardWrapper>
                         <StyledCard height="57.0rem" tabletHeight="auto" padding="2.4rem">
                             <div>
-                                <Header as="h4" type="sub-section-title" mb="0.8rem">
+                                <Header as="h4" type="subtitle-1" mb="0.8rem">
                                     {localize('Revenue share')}
                                 </Header>
-                                <Text>
+                                <Header as="h2" type="paragraph-1" weight="400">
```

Comment:
```suggestion
                                <Header as="h2" type="paragraph-1" weight="regular">
```
Let's use regular for consistency. Few other instances below.